Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,91 @@
+﻿using BootCamp.Chapter.Tests;
+using System;
+using System.Globalization;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileCleaner
+    {
+        /// <summary>
+        /// Cleans up dirtyFileName
+        /// </summary>
+        /// <param name="dirtyFile">Dirty file with "_" placed in random places.</param>
+        /// <param name="cleanedFile">Cleaned up file without any "_".</param>
+        public void Clean(string dirtyFile, string cleanedFile)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'Clean' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)

Reply:
changed it back to static